### ISSUE TITLE
Shuffling without loading a ZIM file keeps on loading without any message

### ISSUE THREAD

**Issue by gaurav7019** [kiwix/kiwix-js#821]
# Bug Report
Clicking on the shuffle button without loading a ZIM file keeps on loading without any error message to the user.

## Current Behavior
Infinite Loading

![127 0 0 1_5500_www_index html (4)](https://user-images.githubusercontent.com/45741026/152675458-d077c082-2d3c-4506-81c4-a1c86a0fc46d.png)

## Expected Behavior
Should throw an alert popup like searching without loading a ZIM file

![127 0 0 1_5500_www_index html (3)](https://user-images.githubusercontent.com/45741026/152675468-81374f1e-6990-42c3-9c02-cf00f28a5838.png)

## Steps to Reproduce
Click on the shuffle button adjacent to the search bar on the homepage without loading a ZIM file

**Comment by gaurav7019** [comment]
Seems like a quick fix so would like to take this up if not a problem, will branch out based on #804 as it would require calling `systemAlert()` implemented in #804

**Comment by Jaifroid** [comment]
Thanks, I confirm this bug.

**Comment by mossroy** [comment]
Thanks for finding this issue. I had not noticed it.

**Comment by gaurav7019** [comment]
@Jaifroid since this was a quick fix, I'm done working on this issue. I have a small doubt though. How should I open a PR in this case? Currently, I created a branch based out of the branch I was already working on (for #804 ). Now, should I rebase it with the master and then create a PR or wait for #804 to merge and then rebase again with the master?

**Comment by Jaifroid** [comment]
@gaurav7019 Since the fix depends on #804, it can't be merged before that one (clearly). I think you would end up with a lot of potentially confusing rebasing if you rebase on master now, and then rebase again after #804. I think it might be easiest to review if based clearly on #804, though I'm not 100% certain how GitHub will present the code.

**Comment by gaurav7019** [comment]
@Jaifroid Should I open a new PR for this from the latest fork to avoid complications?

**Comment by Jaifroid** [comment]
Feel free to -- it's probably the easiest way. Base the new PR on master we've just merged. Thanks!
